core/src/main/scalajvm/sttp/client4/internal/httpclient/WebSocketImpl.scala

```diff
@@ -42,42 +42,27 @@ private[client4] class WebSocketImpl[F[_]](
   override def send(f: WebSocketFrame, isContinuation: Boolean = false): F[Unit] =
     // ws.send* is not thread-safe - at least one can run at a time. Hence, adding a sequencer to ensure that
     // even if called concurrently, these will be run in sequence.
-    sequencer(monad.flatten(monad.eval {
+    sequencer(monad.suspend {
       f match {
         case WebSocketFrame.Text(payload, finalFragment, _) =>
-          fromCompletableFuture(ws.sendText(payload, finalFragment))
+          handleWS(ws.sendText(payload, finalFragment))
```

Comment:
I don't think the name is right here, this method doesn't "handle the websocket" - it just converts the future to the effect, ignoring the result?

Reply:
I changed the name to `fromCompletableFutureToEffect`. Couldn't figured out a better one :/